.gitignore

```diff
@@ -1,3 +1,6 @@
 .idea/
 *.pyc
 *.iml
+logging/
+config_dev.yaml
+passfile.txt
```

Comment:
You don't have to do that. You can write that file into the container if needed. The main point is that configuration is centralized. In the runtime environment, it can be applied however is most convenient / appropriate. I wouldn't compromise security for convenience.